web/src/user_profile.js

```diff
@@ -193,6 +193,11 @@ function render_user_group_list(groups, user) {
 }
 
 function render_manage_profile_content(user) {
+    // Since we want the height of the profile modal to remain consistent when switching tabs,
+    // we need to restrict the height of the main body. This will ensure that the footer of
+    // the "Manage User" tab can adjust within the provided height without expanding the modal.
+    $("#user-profile-modal .modal__body").addClass("modal__body__manage_profile_height");
+    $("#user-profile-modal .manage-profile-tab-footer").addClass("modal__footer_wrapper");
```

Comment:
I feel like there has to be a cleaner solution than this commit for achieving its goals.